.github/workflows/docker.eloqkv.yml

```
@@ -0,0 +1,26 @@
+permissions:
+  contents: read
+  id-token: write
+  attestations: write
+
+jobs:
+  build-eloqkv:
+    uses: ./.github/workflows/docker.template.yml
+    secrets:
+      build-args: |
+        BUILDER_IMAGE=${{ secrets.DOCKER_ORG }}/builder:latest
+      DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
```

actionlint (1.7.10)

[error] secret "DOCKER_USERNAME" is not defined in "./.github/workflows/docker.template.yml" reusable workflow. defined secret is "build-args"

(workflow-call)